### PR TITLE
feat: create using discover services with JS page

### PR DESCRIPTION
**Pull request by stdavis**

### CODE REVIEW

File: _posts/2015-12-21-using-agrcs-new-web-mercator-services-in-your-web-maps.md [hunk 1]
```diff
@@ -13,6 +13,10 @@ categories:
   - Developer
   - Featured
 ---
+<p class="pop text-center">
+  <strong>Note:</strong> Please go to <a href="{% link developer/base-maps/discover/index.md %}">this page</a> for more updated information on using Discover services with the Esri JavaScript API v4.x.
+</p>
+
```
Comment (steveoh) [inline _posts/2015-12-21-using-agrcs-new-web-mercator-services-in-your-web-maps.md:19]: remove click here type link

```suggestion
<p class="pop text-center">
  The instructions on this page are dated. <a href="{% link developer/base-maps/discover/index.md %}">Using Discover services with the Esri JavaScript API v4.x</a> contains more current information.
</p>

```

File: developer/base-maps/discover/index.md [hunk 2]
```diff
@@ -0,0 +1,104 @@
...
+  email: stdavis@utah.gov
+tags:
+  - discover
+  - web services
+categories:
+  - Developer
+date: 2020-04-09 00:00:00
+---
+
+Base map services hosted on discover.utah.gov are delivered via [WMTS](https://en.wikipedia.org/wiki/Web_Map_Tile_Service). WMTS is an [OGC](https://www.ogc.org/) standard for delivering tiled map data. The [Esri JavaScript API](https://developers.arcgis.com/javascript/) provides a class for consuming with these services called a [WebTileLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WebTileLayer.html). Using this class is as simple as passing in the URL to the WMTS service with the appropriate level, column and row tokens. For example:
```
Comment (steveoh) [inline developer/base-maps/discover/index.md:14]:
```suggestion
Base map services hosted on discover.utah.gov are delivered via [WMTS](https://en.wikipedia.org/wiki/Web_Map_Tile_Service). WMTS is an [OGC](https://www.ogc.org/) standard for delivering tiled map data. The [Esri JavaScript API](https://developers.arcgis.com/javascript/) provides a class for interacting with these services, called a [WebTileLayer](https://developers.arcgis.com/javascript/latest/api-reference/esri-layers-WebTileLayer.html). Using this class is as simple as passing in the URL to the WMTS service with the appropriate level, column, and row tokens. For example:
```

File: developer/base-maps/discover/index.md [hunk 3]
```diff
@@ -0,0 +1,104 @@
...
+  const view = new MapView({
+    container: "viewDiv",
+    map: map,
+    center: [-111.5, 40],
+    zoom: 11
+  });
+});
+```
+
+You’ll notice that the URL to the service includes a quad word that is unique to your organization and project. If you don’t have an account set up or a quad word, you can [register on our website]({% link discover/license/index.md %}) to get started. **If you are planning on putting your quad word into a website or somewhere that it is publicly accessible, you will need to {% include contact.html subject=page.title contact=site.data.contacts.discover hide-punctuation=true text='contact' %} to receive a special quad word for licensing reasons.**
```
Comment (steveoh) [inline developer/base-maps/discover/index.md:40]: someone tried to register for a quad word by signing up for the newsletter...

```suggestion
You’ll notice that the URL to the service includes a quad word that is unique to your organization and project. If you don’t have a quad word, you can find the links to [register on our website]({% link discover/license/index.md %}). **If you are planning on using your quad word in a website or a publicly accessible place, you will need to {% include contact.html subject=page.title contact=site.data.contacts.discover hide-punctuation=true text='contact' %} to receive a special quad word for licensing reasons.**
```

**Comment by stdavis** [comment]
@steveoh This is ready for another look if you want to.

**Pull request merged**